src/test/java/org/folio/circulation/resources/RenewalValidatorTest.java

```diff
@@ -18,35 +18,34 @@
 
 public class RenewalValidatorTest {
   @Test
-  public void shouldAllowRenewalWhenDueDateIsEarlierOrSameForDeclaredLostItem() {
+  public void shouldDisallowRenewalWhenDueDateIsEarlierOrSame() {
     val dueDate = now(UTC);
     val proposedDueDate = dueDate.minusWeeks(2);
-    val loan = loanWithItemInStatus("Declared lost", dueDate);
+    val loan = createLoan(dueDate);
 
     val validationResult = errorWhenEarlierOrSameDueDate(loan, proposedDueDate);
 
-    assertTrue(validationResult.succeeded());
+    assertTrue(validationResult.failed());
+    assertThat(validationResult.cause(), instanceOf(ValidationErrorFailure.class));
+
+    val validationFailure = (ValidationErrorFailure) validationResult.cause();
+    assertThat(validationFailure.hasErrorWithReason("renewal would not change the due date"),
+      is(true));
```

Comment:
Could this be extracted into a matcher to reduce the amount of lower level checks in this test?

Reply:
Done, extracted to `hasValidationError` matcher.